### DISCUSSION TITLE
Command line to pull ubuntu-latest build image from github to somewhere else on the Internet ?

### DISCUSSION THREAD

**Discussion by dlmiles** [actions/virtual-environments#7433] [General questions]
What is the pull URL to fetch down the github action image for ubuntu-latest from a container registry ?

I am expecting something like this: docker pull ghcr.io/NAMESPACE/ANYTHINGHERE/ubuntu:latest ?

All the documentation I have seen just uses NAMESPACE placeholder.
This project looks like it manages the contents of the image.

Thanks

**Comment by TimoKramer** [comment]
That would be really interesting

**Comment by mikhailkoliada** [comment]
Hello! We do not provide any containers, only virtual machines in azure

**Reply by TimoKramer** [reply]
thanks, understood

**Comment by dlmiles** [comment]
Just to say over at AWS they have their AMI concept for which vendors can publish, catalogue, solicit gate-keeping for VM images.  So being packaged as a VM should not be a barrier to providing an equivalent URL.

Maybe there is a link to the builder project that creates the VM, in particular it would be a more verbatim form of providing a `Software Bill Of Materials`, with links back to the specific binary origin and source code as and when required by the OSS licensing terms of the individual packages.  That would still be a useful reply allowing regeneration and auditing at the customers cost on Azure.

The reply has been interpreted to read, the VM image is an unauditable secret at this time.
